examples/preprocessing.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Should we crop these logs from the notebook show it becomes a bit more readable?</p><p> </p><p>Something like:</p><p> </p><p>These tasks will be executed: diagnostic_daily/tasmin, diagnostic_daily/script, diagnostic_daily/rsds, diagnostic_daily/tasmax, diagnostic_daily/tdps, diagnostic_daily/tas, diagnostic_daily/vas, diagnostic_daily/pr, diagnostic_daily/uas </p><p>Running 9 tasks sequentially </p><p>Maximum memory used (estimate): 0.2 GB</p><p>Successfully completed tasks</p><p>Note: Log for forcing generation abbreviated for clarity</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I made issue #68 for this.